crates/cargo-winrt/src/main.rs

```diff
@@ -137,6 +158,22 @@ impl Build {
         install.perform()?;
         cargo::build()
     }
+
+    fn print_help() -> anyhow::Result<()> {
```

Comment:
Same here.

Reply:
Well, if we were using `clap`, obviously this would all just be magically done for us. We could do the same kind of attribute/description magic to generate these messages, or at least separate it out into a function with a template string we could call with the list/descriptions and just have it do the output.

As above, I erred on the side of simplicity for this initial change, but a case could certainly be made that now it's TOO simple. :)

Reply:
I like simple. 😉 And I'd rather not bring in additional dependencies so this is fine. We can always revisit if it gets more complicated.